backend/tests/test_health.py

```
@@ -0,0 +1,16 @@
+import sys
+from pathlib import Path
+
+from fastapi.testclient import TestClient
+
+backend_path = Path(__file__).resolve().parents[1] / "app"
+sys.path.append(str(backend_path.parent))
+
+from app.main import app  # type: ignore  # noqa: E402
```

Ruff (0.13.2)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)